### DISCUSSION TITLE
post/202410_non_maximum_suppression/

### DISCUSSION THREAD

**Discussion by giscus** [mistasse/mistasse.be#2] [Announcements]
# post/202410_non_maximum_suppression/

The concept of Non-Maximum Suppression (NMS) is pervasive in Deep Learning solutions, yet I have found it to be rarely discussed, and even less formalized. This is an attempt at demystifying and formalizing it as well as I can. This is an undertaking I have already have started in my thesis, Section 2.2.5, but now is an opportunity to make it more accessible.
Basically, NMS is useful when we want to obtain a sparse set of outputs from a model architecture that, for one or more reasons, does not do it naturally.

https://mistasse.be/post/202410_non_maximum_suppression/
